include/Benchmark/PointsToBenchmark.h

```diff
@@ -0,0 +1,85 @@
+#ifndef POINTS_TO_BENCHMARK_H_
+#define POINTS_TO_BENCHMARK_H_
+#include "Benchmark.h"
+#include "llvm/IR/Metadata.h"
+#include "llvm/IR/DebugInfoMetadata.h"
+
+namespace spatial {
+
+enum Soundness {
+  Sound,
+  Unsound,
+};
+class PointsToBenchmarkRunner : public BenchmarkRunner {
+  Soundness SoundOrUnsound;
```

Comment:
SoundOrUnsound, can we rename this? Is it required to have it this way? How about, bool IsSound?

Reply:
Sure, we can change this. We just created an enum for readability.